packages/js/src/settings/routes/templates/taxonomy.js

```diff
@@ -26,7 +27,8 @@ const FormikReplacementVariableEditorFieldWithDummy = withFormikDummyField( Form
  * @param {string[]} postTypes The connected post types.
  * @returns {JSX.Element} The taxonomy element.
  */
-const Taxonomy = ( { name, label, postTypes: postTypeNames } ) => {
+// eslint-disable-next-line complexity
+const Taxonomy = ( { name, label, postTypes: postTypeNames, showUI } ) => {
```

Comment:
Pretty nit-picky of me, but we don't usually capitalize acronyms, only to get the camel casing for example. I.e. `showUi` in this case?

Reply:
fixed